search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalApiResponseBuilder.kt

```diff
@@ -0,0 +1,241 @@
+package com.egm.stellio.search.web
+
+import com.egm.stellio.search.model.TemporalEntitiesQuery
+import com.egm.stellio.search.model.TemporalQuery
+import com.egm.stellio.shared.model.CompactedAttribute
+import com.egm.stellio.shared.model.CompactedEntity
+import com.egm.stellio.shared.model.toFinalRepresentation
+import com.egm.stellio.shared.util.JsonUtils.serializeObject
+import com.egm.stellio.shared.util.buildQueryResponse
+import com.egm.stellio.shared.util.parseRepresentations
+import com.egm.stellio.shared.util.prepareGetSuccessResponseHeaders
+import com.egm.stellio.shared.util.toHttpHeaderFormat
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.util.MultiValueMap
+import java.time.ZonedDateTime
+
+typealias CompactedTemporalAttribute = List<Map<String, Any>>
```

Comment:
don't really like it because not all compacted temporal attributes look like this